src/features/completions.zig

```diff
@@ -501,19 +501,20 @@ fn populateSnippedCompletions(
     allocator: std.mem.Allocator,
     completions: *std.ArrayListUnmanaged(types.CompletionItem),
     snippets: []const snipped_data.Snipped,
-    config: Config,
+    server: *Server,
```

Comment:
`server` should be the first parameter to be more consistent with all other functions.

Reply:
done.